src/binaryapi/TradingTimes.js

```diff
@@ -22,7 +23,9 @@ class TradingTimes {
     }
 
     async initialize() {
-        await this._serverTime.init(this._api);
+        await this._serverTime.init(this._api, () => {
+            if (typeof this.timeUpdateCallback === 'function') this.timeUpdateCallback();
```

Comment:
I think You can use the new optional chaining operator instead of this if statement:

```suggestion
           this.timeUpdateCallback?.();
```

Reply:
@mahdiaryayi-fs mentioned above